src/Helpers/AsBackgroundJobs.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Give\Helpers;
+
+use ActionScheduler;
+
+/**
+ * @see https://actionscheduler.org/
+ *
+ * @unreleased
+ */
+class AsBackgroundJobs
```

Comment:
Can we find another place to put this instead of the generic helpers file?  Maybe we add a new folder in framework called `ActionScheduler` where we put stuff like this.

Reply:
Resolved in 75665bf1789d8c7801602105ea20f3b896b05c43 and 0efdd386c483557d1cd5f41ce61b2e565158947f